packages/core-components/src/layout/ProxiedSignInPage/ProxiedSignInPage.tsx

```diff
@@ -32,7 +32,7 @@ import { ProxiedSignInIdentity } from './ProxiedSignInIdentity';
  */
 export type ProxiedSignInPageProps = SignInPageProps & {
   /**
-   * The provider to use, e.g. "gcp-iap" or "aws-alb". This must correspond to
+   * The provider to use, e.g. "gcp-iap" or "awsalb". This must correspond to
```

Comment:
Could you add a `patch` changeset for core-components saying `Updated ProxiedSignInPageProps docs` so that this change get released :)